test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/DnxSdk.cs

```diff
@@ -28,7 +28,8 @@ public class DnxSdk
 
         public static string GetRuntimeHome()
         {
-            var homePath = Environment.ExpandEnvironmentVariables(Environment.GetEnvironmentVariable("DNX_HOME"));
+            var dnxHomePath = Environment.GetEnvironmentVariable("DNX_HOME");
```

Comment:
DNX_HOME does not exist on non-Windows platforms so we need to use HOME instead.